src/test/java/ConsoleIOTest.java

```diff
@@ -1,2 +1,44 @@
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
 public class ConsoleIOTest {
+
+    ConsoleIO consoleFixString;
+    ConsoleIO consoleFixNumber;
+    ConsoleIO consoleFixBoolean;
+
+    @Test
+    public void getStringInputCallsGetInput() {
+        ConsoleIOSpy consoleIOSpy = new ConsoleIOSpy(System.in, System.out);
```

Comment:
Might not want to have system in or out within your tests. Could be adding noise to your tests.